doc/examples/reconst_fwdti.py

```diff
@@ -0,0 +1,174 @@
+"""
+=====================================================================
+Title
```

Comment:
Sorry, this got lost down my inbox and discovered this getting back to thinking about this project again. Yes it's still on the horizon, hopefully in a week or two.